shared/folders/index.js

```diff
@@ -58,10 +59,30 @@ class Folders extends Component<void, Props, State> {
   }
 }
 
+// Set rekey meta if this tlf is part of the rekey set
+const injectMeta = (tlf, lockedTLFs) => ({
```

Comment:
the dumb component assumes the meta is 'rekey' if it needs rekey so we're merging the unlock part of the store and the favorites part of the store and changing the meta if its in the unlock store